Security-Operations/L4-Security-Analyst-Technical-Lead.md

```diff
@@ -0,0 +1,87 @@
+
+</details>
+
+## 🌱 Teaching and Mentoring
+
+- I define, document, and evolve best practices in my team's area of focus.
+- I establish training programs, workshops, or knowledge-sharing sessions that uplift the technical technical capabilities of my team.
+- I am comfortable with challenging and mentoring senior and junior analysts, guiding them to take on more responsibility and complex tasks.
+- I help people determine the most valuable ways they can contribute and grow, and I help them block out the noise.
+- I lead by example, demonstrating best practices and high standards in all aspects of my work.
+
+<details>
+<summary>Examples</summary>
+
+- I developed and led a comprehensive training program for the Security Operations team, covering advanced topics like threat intelligence and malware analysis.
+- I broke up a project in such a way that I lined up appropriate challenges for each of my teammates, which helped them grow.
+- I had some difficult conversations with my teammates, challenging them directly while showing them my care for them personally.
+- I mentored a more junior analyst through the process of obtaining an advanced cybersecurity certification, which they successfully achieved.
+- I worked closely with my manager to create a career development plan for a more junior security analyst, helping them understand their growth paths and required skills.
+- I developed a playbook for handling security incidents that has since become a go-to resource for new hires and seasoned engineers.
+
+</details>
+
+## 🧭 Culture and Leadership
+
+- I encompass all of the qualities listed in [Leadership](https://github.com/OctopusDeploy/People/blob/main/Leadership.md).
+- I avoid becoming a bottleneck by setting up my team for success for longer periods before needing to check back in.
+- I ensure that difficult work, such as troubleshooting, security incident response, and threat hunting, gets done.
+- I understand our organisational structure and how we schedule work, and I help my team find their highest-value contributions in this environment.
+- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately, and dealing with the consequences along the way.
+- I recognise problems and proactively engage with other people to prevent or resolve them quickly.
+- I am comfortable coordinating projects where success requires the input of multiple teams and diverse technical roles.
+- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
+
+<details>
+<summary>Examples</summary>
+
+- I identified the work involved with delivering a Pitch or RFC, broke it down into tasks, and managed the project to completion.
+- I led a post-mortem after a security incident, facilitating an open and blame-free discussion that resulted in actionable improvements to our security posture.
```

Comment:
I'm working with @cailyoung at the moment on Incident Management. One aspect where we see a gap is going from localised learning to global learning - I wonder if it's too early to start influencing right now, but let's earmark time in the next 6 months to update all the roles with this trait: AND helped other people across Octopus learn from the incident.